taxi/models.py

```diff
@@ -15,7 +16,11 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=255,
+        unique=True,
+        validators=[LicenseValidator()]
```

Comment:
Ensure that the `LicenseValidator` is correctly implemented in the `validators` module to validate the `license_number` field as expected.